bundles/org.openhab.ui/web/src/components/cards/glance/location/status-badge.vue

```diff
@@ -42,6 +42,7 @@ export default {
     return {
       badgeConfigs: {
         alarms: { icon: 'f7:exclamationmark_triangle_fill' },
+        battery: { icon: 'oh:lowbattery', state: 'on' },
```

Comment:
I prefer `f7:battery_25` to `oh:lowbattery` (monochrome and more representative I think:

![image](https://user-images.githubusercontent.com/2004147/119055580-96881d80-b9c9-11eb-97d0-7fe4c261b56a.png)

![image](https://user-images.githubusercontent.com/2004147/119055652-b28bbf00-b9c9-11eb-9267-eedcf2a55d0f.png)